lib/l10n/app_en.arb

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
This file is still there?

Reply:
We delete that on the previous PR, It's there on the existing PRs, I'll delete it after

Reply:
Okay, Please don't forget to do it.